26.md

```diff
@@ -0,0 +1,80 @@
+NIP: 26
+=======
+
+Delegated Event Signing
+-----
+
+`draft` `mandatory` `author:markharding` `author:minds`
```

Comment:
only nip01 is mandatory, some relays may not want to implement this and that's ok.

Reply:
My concern is that all of the events coming out of Minds will be invalid to *most* clients and rejected by relays too.

Reply:
had some comments on this here:

https://github.com/nostr-protocol/nips/pull/28#issuecomment-1205659610